tests/testsuite/git_status.rs

```diff
@@ -259,7 +239,6 @@ fn doesnt_show_untracked_file_if_disabled() -> io::Result<()> {
 }
 
 #[test]
-#[ignore]
```

Comment:
Not sure why these were ignored (couldn't tell from history). Ok to un-ignore?

Reply:
I'd prefer we didn't. Anything that touches the filesystem should be ignored in the tests (in practice, we use `#[ignore]` as a proxy for 'don't run on user machines'), because any user looking to do a regular install may run them.

We've already had one bug where (improperly) non-ignored git tests overwrote a bunch of people's git configs with nonsense data.

Reply:
@chipbuster Added back the `#[ignore]`'s